### ISSUE TITLE
File labeling for domain and problem

### ISSUE THREAD

**Issue by AnjelicaB** [AI-Planning/planutils#109]
Inside the Planutils docker environment, I want to run prp with my domain and problem files with the comand: (`(planutils) root@...:~$ prp [DOMAIN_FILE] PROBLEM_FILE SEARCH_OPTION`) But how do I access the domain and problem files? When I'm in the base command line, normally I could just type in the path name of those files, but how does it work in the planutils environment? For context, I'm a Mac user.

**Comment by haz** [comment]
This is a docker question, and not something specific to planutils. You could checkout files from inside the container, but probably the more natural approach is to use a virtual mount. Something like this...

```bash
docker run -it --privileged -v $(pwd):/PROJECT aiplanning/planutils
```

That way, the local folder (i.e., wherever you run the above command) will be mounted inside the container at the `/PROJECT` directory.

**Comment by haz** [comment]
You'll need to place the pddl in the home directory of the docker container (so `/root`). Any subfolder there would be fine too.

**Comment by AnjelicaB** [comment]
Ok I managed to solve this problem by first creating the docker container with `docker build -t planutils:latest`. I exited out of the container, and copied the domain and problem files into the root of the container with `docer cp <path name to the file> <id of docker container>:./root`, and then run that container `docker exec -t <id of container> /bin/bash`. Perhaps there's another way but this is how I did it and it worked!
